src/components/SummaryStats/SummaryStats.style.tsx

```diff
@@ -74,9 +75,18 @@ export const SummaryStatWrapper = styled(Box)<{ condensed?: Boolean }>`
 
     @media (min-width: 600px) {
       border-bottom: 0;
-      padding: 0;
+      margin: 0;
+      padding: 0 1.2rem 0 1rem;
       flex-direction: column;
-      align-items: center;
+      align-items: flex-start;
```

Comment:
I think you need `align-items: center`, the stats look a bit off with `flex-start`

![image](https://user-images.githubusercontent.com/114084/82136810-395df880-97c6-11ea-83f1-95e2b5ddad7d.png)

Reply:
cool i think @adpirz was followig a new mock. but i also flagged this so maybe we raise as feedback to UX?